docs/src/types/evm_address.md

```diff
@@ -0,0 +1,15 @@
+# EvmAddress
+
+In the Rust SDK, evm addresses can be represented with the 'EvmAddress' type. Its definition matches with the Sway standard library type with the same name and will be converted accordingly when interacting with contracts:
+
+```rust,ignore
+{{#include ../../../packages/fuels-core/src/types/bits_256.rs:evm_address}}
+```
+
+Here's an example:
+
+```rust,ignore
+{{#include ../../../packages/fuels/tests/bindings.rs:evm_address_arg}}
+```
+
+> **Note:** when creating an `EvmAddress` from `Bits256`, the first 12 bytes will be cleared because an evm address is only 20 bytes long.
```

Comment:
Explain the choice of clearing the leftmost ones. If it is a deliberate choice (ie not based on spec), then this choice should be stated clearly ("We chose the convention of clearing the leftmost bytes").

Reply:
The choice was made based on the Sway implementation of the same type. I find this a bit tricky to comment since we had a task to specifically remove all mentions of sway.